CRM/Core/Form/Search.php

```diff
@@ -85,6 +85,12 @@ protected function buildTaskList() {
     return $this->_taskList;
   }
 
+  /**
+   * Should we be adding all the metadata for contact search fields or just for the sort name
```

Comment:
Non blocking but the standard is a full stop & then a blank line before the var declaration